coap/adapter.go

```diff
@@ -99,7 +126,7 @@ func (svc *adapterService) Unsubscribe(clientID string) {
 func (svc *adapterService) SetTimeout(clientID string, timer *time.Timer, duration int) (chan bool, error) {
 	sub, ok := svc.get(clientID)
 	if !ok {
-		return nil, errEntryNotFound
+		return nil, errors.New("observer entry not founds")
```

Comment:
Fix typo. Should this error be public?

Reply:
No need for that. It wasn't exported before, and this its only usage.